### PR TITLE
Issue 773/Migrate ScrollableCard to MUI5

### PR DESCRIPTION
**Pull request by aharpt**
https://www.loom.com/share/44ec39e7e86a4acaa49c76b6236772d8

### CODE REVIEW

File: src/components/SQFormScrollableCard/SQFormScrollableCard.tsx [hunk 1]
```diff
@@ -74,58 +66,10 @@ export type SQFormScrollableCardProps<Values extends FormikValues> = {
   validationSchema?: AnyObjectSchema;
   /** Boolean used to determine if title/header is enabled or disabled */
   isHeaderDisabled?: boolean;
-  /** MUI Typography variant to be used for the title */
-  titleVariant?: TypographyVariant;
```
Comment (aharpt) [inline src/components/SQFormScrollableCard/SQFormScrollableCard.tsx:78]: This prop was not used anywhere in the component so I just deleted it.

File: src/components/SQFormScrollableCard/SQFormScrollableCard.tsx [hunk 2]
```diff
@@ -1,26 +1,18 @@
...
-import type {TypographyVariant, GridProps} from '@material-ui/core';
+import type {GridProps} from '@mui/material';
 import type {FormikHelpers, FormikValues} from 'formik';
 import type {AnyObjectSchema} from 'yup';
 
 export type SQFormScrollableCardProps<Values extends FormikValues> = {
-  /** An object of css-in-js style properties to be passed and spread onto `classes.cardContent` */
-  cardContentStyles?: CreateCSSProperties;
+  /** An object of css-in-js style properties to be passed and spread onto the `CardContent` component */
+  cardContentStyles?: Record<string, unknown>;
```
Comment (laurelbean) [inline src/components/SQFormScrollableCard/SQFormScrollableCard.tsx:15]: I think we should use the `SxProps` type for these. 
`import { SxProps } from '@mui/material'`
or if that doesn't work, then try 
`import { SxProps } from '@material-ui/system';`

found here: https://github.com/mui/material-ui/issues/27564

Comment (20BBrown14) [inline src/components/SQFormScrollableCard/SQFormScrollableCard.tsx:15]: We should be slight more lenient with our typing here. I think we should crib exactly whatever the `sx` prop is. I think it also accepts an array? Is there a particular reason some variation of CSS PRoperties wouldn't work?

Comment (aharpt) [reply src/components/SQFormScrollableCard/SQFormScrollableCard.tsx:15]: Yeah that's a better idea.  I'll change that.

Comment (aharpt) [reply src/components/SQFormScrollableCard/SQFormScrollableCard.tsx:15]: @20BBrown14 I am working on trying to implement @laurelbean's suggestion.  I couldn't find the `CreateCSSProperties` type in MUI5 which is the reason for the change.

Comment (aharpt) [reply src/components/SQFormScrollableCard/SQFormScrollableCard.tsx]: @laurelbean @20BBrown14 Here is an updated loom outlining the solution I came to and why I went that direction.  Feel free to take a look if interested. https://www.loom.com/share/41cc6dc460994552ad3354c56c3c9690

**Pull request merged**